services/frontend-service/src/ui/components/ServiceLane/ServiceLane.test.tsx

```diff
@@ -111,6 +111,16 @@ describe('Service Lane', () => {
     });
 });
 
+const getMockRelease = (version: number): Release => ({
```

Comment:
suggestion: rename to `makeReleaseMock` or `makeRelease`

Reply:
:+1: